tools/godotcpp.py

```diff
@@ -545,15 +558,12 @@ def _godot_cpp(env):
         env.NoCache(bindings)
 
     # Sources to compile
-    sources = []
-    add_sources(sources, "src", "cpp")
-    add_sources(sources, "src/classes", "cpp")
-    add_sources(sources, "src/core", "cpp")
-    add_sources(sources, "src/variant", "cpp")
+    sources = env.Glob("src/*.cpp")
+    sources += env.Glob("src/*/*.cpp")
```

Comment:
I'm not sure why the globs are being touched; I agree it makes sense to de-explicitize if we're just globbing anyway, but it doesn't really pertain to the PR.

Reply:
`add_sources` can be removed for `Glob`, but I decided to use the current explicit globs instead of an "all subdirectories" glob to be more agnostic to that decision.